### DISCUSSION TITLE
[Fixed in 1.76.0] Anchor links redirect to the top, bug?

### DISCUSSION THREAD

**Discussion by mcanouil** [community/community#29212] [Mobile] [answered]
When browsing a README.md file which includes a table of contents (e.g., any Awesome List), clicking on any links from the TOC does not follow to the proper section, instead it goes straight to the top of the README.md

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.75.0 (74400986)
Device: iPhone 8 (iOS 15.6)
</details>

**Comment by radar-blip** [comment]
A Table of Contents that you made? or any Table of Contents?

**Reply by mcanouil** [reply]
Any TOC written as markdown, e.g., https://github.com/sindresorhus/awesome
The TOC is working just fine elsewhere, but not on the mobile app.

Btw and since you seem to imply there is a TOC not "hand made", what other kind of TOC, that you can use on the mobile app, exist? The top left button for the GitHub TOC does not exist in the mobile app.

**Reply by radar-blip** [reply]
I was not implying anything, just asking if it was code you made, or if the problem is occuring everywhere. 
Anyway, I did not catch that this was about the mobile app, sorry for the confusion.

**Reply by hwdsl2** [reply]
I observed the same issue - html anchor links are broken in the mobile app. They jump to top of page instead of the correct location.

GitHub please investigate and fix.

**Reply by mcanouil** [reply]
I opened an issue to GitHub support.
They are working on it in theory according to the answer I got from them.

**Comment by mcanouil** [comment, marked as answer]
This has been fixed in 1.76.0
